index.html

```diff
@@ -1,228 +1,326 @@
-                    <hr>
-                    <p>Greater Washington Urban League (Thursday Network)
-                        <br> 2901 14th Street, NW
-                        <br> Washington, D.C., 20024 </p>
-                    <p><i class="fa fa-envelope" aria-hidden="true"></i> <a href="mailto:ppd@thursdaynetwork.org">ppd@thursdaynetwork.org</a></p>
-                    <hr>
-                    <p class="lead">To find out about sponsoring this event or making a donation, please reach out to Amy Lee Walton, the Personal and Professional Development chair of Thursday Network and Program Manager of this event at</p>
-                    <p><i class="fa fa-envelope" aria-hidden="true"></i> <a href="mailto:ppd@thursdaynetwork.org">ppd@thursdaynetwork.org</a></p>
-                </div>
-            </div>
-        </div>
-    </section>
-
-    <!-- FOOTER -->
-
-    <footer id="footer">
-        <div class="container">
-            <div class="row text-center">
-                <div class="col-lg-12">
-                    <p>Copyright © 2017 Thursday Network - <a href="https://www.thursdaynetwork.org" target="_blank">thursdaynetwork.org</a> | All right reserved.</p>
-                </div>
-            </div>
-        </div>
-    </footer>
-
-    <!-- JS -->
-
-    <script src="js/jquery.js"></script>
-
-    <script src="js/bootstrap.min.js"></script>
-    <script src="js/jquery.nav.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-countdown/2.0.1/jquery.plugin.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-countdown/2.0.1/jquery.countdown.min.js"></script>
-    <script src="https://rawgit.com/aterrien/jQuery-Knob/master/dist/jquery.knob.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.2/moment-with-locales.min.js"></script>
-    <script src="js/wow.min.js"></script>
-    <script src="js/main.js"></script>
-</body>
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Thursday Network: Hackathon 2017</title>
+    <meta name="description" content="">
+
+    <!-- FONTS -->
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,500,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600" rel="stylesheet">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+
+    <!-- CSS -->
+    <link href="https://api.mapbox.com/mapbox-gl-js/v0.38.0/mapbox-gl.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/animate.css">
+    <link rel="stylesheet" href="css/main.css">
+
+    <!-- JS - Important to load this in head -->
+    <script src="https://api.mapbox.com/mapbox-gl-js/v0.38.0/mapbox-gl.js"></script>
```

Comment:
@impreze When placing that script down on line 313, the browser doesn't render the map at all (see image below). Whereas placing this script in the head does render the map.

 ![mbapi](https://user-images.githubusercontent.com/17731291/27565946-8a30f322-5aaf-11e7-951e-0eedff2a4951.jpeg)